aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -56,4 +56,40 @@ Tensor flip_cpu(const Tensor& self, IntList dims) {
   return out_tensor;
 }
 
+Tensor rot90(const Tensor& self, int64_t k, IntList dims) {
+  const int64_t total_dims = self.dim(), total_rot_dims = dims.size();
+
+  AT_CHECK(total_rot_dims == 2,
+    "expected total rotation dims == 2, but got dims = ", total_rot_dims);
+
+  AT_CHECK(dims[0] != dims[1],
+    "expected rotation dims to be different, but got both dims = ", dims[0]);
+
+  // check range of dims
+  auto rot_dims_v = std::vector<int64_t>(dims);
+  std::sort(rot_dims_v.begin(), rot_dims_v.end());
+
+  AT_CHECK(rot_dims_v[0] >= 0,
```

Comment:
Why don't we want to support wrap_dim?

Reply:
because I didn't support wrap_dim in flip()...should I fix this?

Reply:
yes please :)